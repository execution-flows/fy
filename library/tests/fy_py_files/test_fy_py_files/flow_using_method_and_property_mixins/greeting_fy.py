"""fy
property greeting: str using constant:
"""

from functools import cached_property


# fy:start <<<===
class Greeting_UsingConstant_PropertyMixin:
    @cached_property
    def _greeting(self) -> str:
        # fy:end <<<===
        return "Hello world!"
