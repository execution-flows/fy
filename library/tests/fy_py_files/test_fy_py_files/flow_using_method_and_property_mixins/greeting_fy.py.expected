"""fy
property greeting: str using constant:
"""

# fy:start <<<===
class Greeting_UsingConstant_PropertyMixin:
    @property
    def _greeting(self) -> str:
        # fy:end <<<===
        return "Hello world!"
